src/js/common/components/form-elements/GrowableTable.jsx

```diff
@@ -212,12 +248,22 @@ GrowableTable.propTypes = {
   isValidSection: React.PropTypes.func.isRequired,
   addNewMessage: React.PropTypes.string,
   minimumRows: React.PropTypes.number,
-  rowTitle: React.PropTypes.string
+  rowTitle: React.PropTypes.string,
```

Comment:
Just checked and it seems fine. I made a minor tweak to the button style and fixed a validation issue on the review page that I noticed, though.